doc/ref_cert/RC1/chapters/chapter03.rst

```diff
@@ -112,25 +121,28 @@ CNTT <https://git.opnfv.org/functest/tree/docker/smoke-cntt/testcases.yaml?h=sta
 According to :doc:`ref_arch/openstack/chapters/chapter05`
 the following test names must not be executed:
 
-+-------------------------------------------------+--------------+
-| test rejection regular expressions              | reasons      |
-+=================================================+==============+
-| .\*api.identity.v3.test_oauth1_tokens           | oauth1       |
-+-------------------------------------------------+--------------+
-| .\*scenario.test_federated_authentication       | federation   |
-+-------------------------------------------------+--------------+
-| .\*identity.admin.v2                            | API v2       |
-+-------------------------------------------------+--------------+
-| .\*identity.v2                                  | API v2       |
-+-------------------------------------------------+--------------+
-| .\*identity.v3.test_access_rules                | access_rules |
-|                                                 |              |
-|                                                 |              |
-+-------------------------------------------------+--------------+
-| .\*identity.v3.test_application_credentials.\   | access_rules |
-| ApplicationCredentialsV3Test.\                  |              |
-| test_create_application_credential_access_rules |              |
-+-------------------------------------------------+--------------+
+.. list-table:: Keystone API testing
+   :widths: 60 40
+   :header-rows: 1
+
+   * - Test rejection regular expressions
+     - Reasons
+   * - .*api.identity.v3.test_oauth1_tokens
+     - oauth1
+   * - .*scenario.test_federated_authentication
+     - federation
+   * - .*identity.admin.v2
+     - API v2
+   * - .*identity.v2
+     - API v2
+   * - .*identity.v3.test_access_rules
+     - access_rules
+   * - .*identity.v3.test_application_credentials.\\
+
```

Comment:
should these extra lines 141 and 143 be deleted?

Reply:
It's about manual wrapping. I do add a blank line to make it work.